acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/BitbucketBranchSource.java

```diff
@@ -43,4 +68,25 @@ public BitbucketBranchSource repositoryName(String repositoryName) {
         this.repositoryName.set(repositoryName);
         return this;
     }
+
+    private void addTrait(String name) {
+        control(By.cssSelector(".trait-container .trait-add")).click();
+        WebElement traitOption =  control(By.linkText(name)).resolve();
+        if (traitOption.isEnabled()) {
+            runWithRetry(traitOption::click);
+        }
```

Comment:
Should this "close" the list if it already has the option? (which I believe is indicated by the `isEnabled` )

Reply:
Just tested this and clicking on an option that has already been selected (disabled) will actually just close the dropdown so we don't need to explicitly close it.

Reply:
We we don't click if the option is disabled, and I believe that an already present option is "disabled" right?

Reply:
Ah I see what you mean, sorry I missed that. I'll just remove the isEnabled check and just click the option regardless of the state, as it won't do anything if it is disabled anyway.